src/Package/Impl/Repl/VsRInteractiveWorkflowProvider.cs

```diff
@@ -74,10 +79,20 @@ internal class VsRInteractiveWorkflowProvider : IRInteractiveWorkflowProvider, I
                                                     _plotsProvider, _activeTextViewTracker, _debuggerModeTracker, 
                                                     _shell, settings, _wss, () => DisposeInstance(sessionProvider));
             _disposableBag.Add(workflow);
+
+            sessionProvider.BrokerChanging += OnBrokerChanging;
             return workflow;
         }
 
+        private void OnBrokerChanging(object sender, EventArgs e) {
```

Comment:
But why it should steal focus?

---
In reply to: [79254918](https://github.com/Microsoft/RTVS/pull/2341#discussion_r79254918) [](ancestors = 79254918)